libc/string/string_errsig.c

```diff
@@ -48,6 +51,41 @@ static const unsigned int error_codes_len = sizeof(error_codes) / sizeof(error_c
 
 const unsigned int signal_codes_len = sizeof(signal_codes) / sizeof(signal_codes[0]);
 
+
+char *perrorToFile(char *msg)
+{
+	int errnoBefore = errno;
+
+	FILE *file = fopen("error.txt", "w+");
+
+
+	// chmod("error.txt", 0777);
+
+	TEST_ASSERT_NOT_NULL(file);
+	TEST_ASSERT_NOT_EQUAL(-1, dup2(fileno(file), 2));
```

Comment:
From what I remember, there should be a variable defining which file descriptor is for stderr. I know this is likely to be constant, but in case of changes, we are sure that nothing will break.

```suggestion
	TEST_ASSERT_NOT_EQUAL(-1, dup2(fileno(file), STDERR_FILENO));
```